src/framework/components/rigid-body/system.js

```diff
@@ -310,6 +377,9 @@ class RigidBodyComponentSystem extends ComponentSystem {
      * [0, -9.81, 0] which is an approximation of the gravitational force on Earth.
      *
      * @type {Vec3}
+     * @example
+     * // Set the gravity in the physics world to simulate a planet with low gravity
```

Comment:
No need //

Reply:
It's a code comment, so it's needed (in case someone copies and pastes the code block).